sql/plan/resolved_table.go

```diff
@@ -0,0 +1,110 @@
+package plan
+
+import (
+	"fmt"
+	"io"
+
+	"gopkg.in/src-d/go-mysql-server.v0/sql"
+)
+
+// ResolvedTable represents a resolved SQL Table.
+type ResolvedTable struct {
```

Comment:
Does it mean that for `ResolvedTable` we keep also `Node` interface?

Reply:
No, I think you're right resolved table needs a Node, but in pure table we have to get rid of node.

Reply:
yes, you can see a dummy implementation of  `sql.TableV2` in the test used by the `ResolvedTable` node.